yarn.lock

```diff
@@ -303,6 +303,26 @@ ajv@^6.10.0:
     json-schema-traverse "^0.4.1"
     uri-js "^4.2.2"
```

Comment:
I see the `universal-analytics` module introduced quite a lot of new dependencies. Can you please try this one: https://www.npmjs.com/package/universal-ga which seems to have no dependencies?